va/va.h

```diff
@@ -1111,6 +1115,21 @@ typedef union _VAConfigAttribValEncRateControlExt {
     uint32_t value;
 } VAConfigAttribValEncRateControlExt;
 
+/** \brief Attribute value for VAConfigAttribMultipleFrame*/
+typedef union _VAConfigAttribValMultipleFrame {
+    struct {
+        /** \brief max num of concurrent frames from different stream */
+        max_num_concurrent_frames      : 8;
+        /** \brief indicate whether MF support mixed codecs */
+        mixed_codecs                   : 1;
+        /** \brief indicate whether all stream must support same quality level */
+        mixed_quality_level            : 1;
```

Comment:
As as above.

Reply:
describe it , need 7 bits at least, and cant be common, I dont know how much quality levels is supported by other vendors

Reply:
we support any combination of mixed quality level. so will it be ok to just use one flag?